src/connection.js

```diff
@@ -1693,19 +1693,15 @@ class Connection extends EventEmitter {
       let message;
 
       this.request.once('cancel', () => {
-        if (!this.isRequestActive(request)) {
```

Comment:
Wouldn't it be better to have `isRequestActive` also return true if the state is `this.STATE.BUILDING_CLIENT_REQUEST`? 🤔

Reply:
I was thinking that too, but I didn't see a Building Client Request state in the TDS docs so I wasn't too sure. Do you think I should add that as part of the `isRequestActive` method?